README.md

```diff
@@ -297,7 +299,24 @@ either enable "debug" dracut module or add `dracut_install netstat ip` line to
 (for "rdsosreport.sh") show - there can be no default route, whatever interface
 naming mixup, no traffic (e.g. unrelated connection issue), etc.
 
-## 5.4 Report a bug
+## 5.4 *dracut: dropbearconvert for ED25519 key failed*
+
+In the case that your dropbearconvert does not support ED25519 encryption (e.g CentOS Stream), you will need to install
+dropbear from source upstream of [this PR](https://github.com/mkj/dropbear/pull/91).  
```

Comment:
Linking the PR instead of the first release that supported ED25519 seems like the wrong move. Makes it much harder to figure out what version is necessary. Dropbear 2020.79 seems to have been the relevant release.

Using the version also allows us to omit "CentOS Stream" as an explicit example. Hopefully it will be updated at some point (why hasn't it been yet??) and we avoid having an obsolete example in the README.

Reply:
I think, as below, an issue with `dropbear` in `/usr/local/bin` against `dropbear` in `/usr/`  will update this doc.  

> Hopefully it will be updated at some point (why hasn't it been yet??) and we avoid having an obsolete example in the README.
Sure, but that's why this is part of the troubleshooting guide right?